app/_src/gateway/upgrade/lts-upgrade.md

```diff
@@ -0,0 +1,217 @@
+
+By following this dual cluster deployment strategy, you can achieve a smooth and zero-downtime upgrade from one LTS version of Kong to another. This approach helps ensure high availability and uninterrupted service for your users throughout the upgrade process.
+
+#### In-place upgrade
+
+Plan a suitable maintenance or downtime window during which you can perform the upgrade. During this period, the {{site.base_gateway}} will be temporarily unavailable.
+
+While an in-place upgrade deployment allows you to perform the upgrade on the same infrastructure, it does require some downtime during the actual upgrade process. 
+
+For scenarios where zero downtime is critical, you would need to consider the dual cluster deployment method mentioned earlier, but that would involve additional resources and complexities.
+
+### DB-less mode
+
+DB-less mode is special in that each independent Kong node loads a copy of Kong configuration data into memory, without persistent database storage, so failure of some nodes will not spread to others.
+
+Customers in this mode are recommended to adopt the rolling upgrade strategy. This strategy relies on changes being made to the kong.yaml and then reloading the db-less instance of Kong.
+
+It is highly recommended to take a backup of your current `kong.yaml` file before starting the upgrade as mentioned above.
+
+### Hybrid mode
+
+Hybrid mode comprises one or more control plane (CP) nodes, and one or more data plane (DP) nodes. 
+CP nodes use a database to store Kong configuration data, whereas DP nodes do not, since they get all of the needed information from the CP.
+Therefore, the recommended upgrade process is a combination of different upgrade strategies for each type of node, CP or DP.
+
+The major challenge with a hybrid mode upgrade is the communication between CP and DP.
+
+In hybrid mode, the CP can't be running a version of Kong that is older than the versions of the DPs. For 
+this reason, you must upgrade your CPs first, and then the DPs. With this method, there is no business downtime.
+
+See the upgrade workflow in the following diagram.
+
+![Upgrading a hybrid mode deployment](/assets/images/products/gateway/upgrade/hybrid-mode-upgrade.png)
+> _Figure 2: Upgrading a hybrid mode deployment using a dual-cluster or in-place strategy for control planes,_
+_followed by a rolling upgrade strategy for data planes_.
+
+1. First, upgrade the CP according to the recommendations in the section [Traditional Mode](#traditional-mode), while DP nodes are still serving API requests.
+2. Next, upgrade DP nodes using the recommendations from the section [DB-less Mode](#db-less-mode). 
+Point the new DP nodes to the new CP to avoid version conflicts.
+
+## Preparation: Review gateway changes
```

Comment:
I wonder if it would be better to move these to the breaking changes section as a 2.8 LTS to 3.4 LTS breaking changes page of its own. That would be consistent with the other pages.

Reply:
For the "Notifications of changed behavior" subsection, can we just tell them to read the release notes especially if this is just for awareness and not for actionable steps? Just trying to avoid duplicate content/too much content on one page.

Reply:
> I wonder if it would be better to move these to the breaking changes section as a 2.8 LTS to 3.4 LTS breaking changes page of its own. That would be consistent with the other pages.

I would be fine with this, not sure how it would go over though. Let's try it out.

> For the "Notifications of changed behavior" subsection, can we just tell them to read the release notes especially if this is just for awareness and not for actionable steps? Just trying to avoid duplicate content/too much content on one page.

Unfortunately no, for the same reason that I put for your previous comment - the goal of the LTS guide is to consolidate all the info they might need, and reading through the changelog is far more difficult.